File: backend/src/Modules/Modules.Artwork.Application/Artworks/GetAll/ArtWorkListItem.cs

namespace Modules.Artwork.Application.Artworks.GetAll;
public sealed record ArtworkListItem(
    Guid Id,
    string TitleHu,
    string? TitleEn,
    int? Year,
    string? TechniqueHu,
    string? TechniqueEn,
    decimal? WidthCm,
    decimal? HeightCm,
    Guid? ImageId,
    bool IsFeatured,
    int DisplayOrder);

File: frontend/henn-laszlo/projects/artwork-data-access/src/lib/models/artwork-list-item.ts
export interface ArtworkListItem {
  readonly id: string;
  readonly titleHu: string;
  readonly titleEn: string | null;
  readonly year: number | null;
  readonly techniqueHu: string | null;
  readonly techniqueEn: string | null;
  readonly widthCm: number | null;
  readonly heightCm: number | null;
  readonly isFeatured: boolean;
  readonly displayOrder: number;
}